### ISSUE TITLE
Command Line Interface

### ISSUE THREAD

**Issue by JindalAman1610** [andreas-mausch/whatsapp-viewer#73]
Do you have or could provide a cmd interface in order to extract and export the chats directly?
Something which could work using a script?

**Comment by andreas-mausch** [comment]
Agree, would be the next thing I'd do. However I wasn't really active on this project for a while, so I can't give an ETA.
